package.json

```diff
@@ -10,6 +10,7 @@
     "gulp-plumber": "^0.6.6",
     "gulp-prettify": "^0.3.0",
     "gulp-s3": "^0.3.0",
+    "gulp-sitemap": "^2.0.1",
```

Comment:
heh that's cool, didn't know there was a plugin that automates this for us!!

neat.